fairscale/utils/testing.py

```diff
@@ -54,7 +61,25 @@ def set_random_seed(seed: int) -> None:
     model_parallel_cuda_manual_seed(seed)
 
 
-def dist_init(rank, world_size, hostname=None):
+def torch_version() -> Tuple[int, ...]:
+    result = version.parse(torch.__version__).release
```

Comment:
Nice fix. Maybe remove the dependency on packaging (always nice to remove dependencies). That way there is only one way we do this.